src/Pages/RequestPage/Component/RequsetList.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component, Fragment } from "react";
+import styled from "styled-components";
+
+class RequsetList extends Component {
+  state = {
+    requestList: [],
+  };
+
+  componentDidMount() {
+    fetch("Data/userData.json")
+      .then((res) => res.json())
+      .then((result) => this.setState({ requestList: result }));
```

Comment:
```suggestion
      .then(({ requestList }) => this.setState({ requestList }));
```
이렇게도 가능은 합니다. (참고 : 객체 단축 속성명)

Reply:
수정했습니다!